pom.xml

```diff
@@ -200,6 +200,20 @@
             <version>0.2.0</version>
         </dependency>
 
+        <dependency>
```

Comment:
Logging is added via Spring-boot. What does this add that Spring-boot does not?

Reply:
Codacy was getting logging binding errors so I thought they were the cause of the problem but they that was not the case. I removed it now.